### ISSUE TITLE
Modified Base Output

### ISSUE THREAD

**Issue by fneuh** [nanoporetech/taiyaki#31]
Hi,
I trained a modification-aware RNA model, the modification that I looked at, can occur only at G's. My training dataset consists of reads that are either completely modified (so every G has the modification) or completely unmodifed.
After training I basecalled a held-out test set, that also consists of reads that are completely modified or unmodified, with _basecall.py_
When looking at the modifed base scores, modified positions have a modified base score from -2 to 0, and unmodified positions have a score from around -20 to -2. There is a bit of intercept between these populations, but over all the distributions of unmodified and modified reads are separated quite clear. In docs/FILE_FORMATS you write:

> More negative modified base scores indicate modified bases and more positive scores indicate canonical bases.

So that's more or less the opposite behavior of what I would expect from looking at my data..
Do you have an idea, where this problem is coming from? Maybe by any luck there is a mistake in the file-format doc? 

Thanks,
Florian

**Comment by marcus1487** [comment]
This is indeed a mistake in the docs. The score is approximately the log likelihood that a base is modified (though in practice these values might not match empirical likelihoods). Thus higher scores (closer to 0) indicate that a base is modified and lower scores indicate lower probability that a base is modified. We will update the docs accordingly.

**Comment by fneuh** [comment]
Ok :)
Thanks for the quick reply!
